group-generators/generators/talentlayer/index.ts

```diff
@@ -176,11 +176,11 @@ const generator: GroupGenerator = {
     const usersGroup = await generateUsersGroup(context);
     const didServiceGroup = await generateDidSellerServiceForBuyerGroup(
       context,
-      "alice",
```

Comment:
There will be some issue regarding your group generation because the GitHub Repo names:  "TalentLayer/talentlayer-id-contracts",
"TalentLayer-Labs/indie-frontend",
"TalentLayer/talentlayer-id-subgraph",
are not correct anymore